pkg/collector/corechecks/system/wincrashdetect/probe/README.md

```diff
@@ -0,0 +1,114 @@
+## How this works, and how to test.
+
+### How this works
+
+Windows provides a limited subset of the debugger functionality with the base OS.  This allows us to execute some (not all) debugger commands when we discover a crash dump file.  But, the output is very
+limited.  It simply dumps the strings that would appear in the debugger.  So we have to parse the string
+to find the parts we're interested.  A sample output is supplied below.
+
+### How to (really) test
+
+Unfortunately, actually testing the parser then requires actual dumps.  Actual dumps require multiple gigabytes of data.  To manually test, in a VM (don't do this on your own machine), install the crasher driver (TBD: linked here).  Start the driver. It will _immediately_ crash.  On reboot there will be a crash dump to parse, and (assuming the agent is installed) the agent will find and report.
```

Comment:
`TBD: linked here`
Not sure if it's for a future PR or just forgotten

PS: you could write an e2e test with the new framework to test that !